tap_klaviyo/__init__.py

```diff
@@ -5,37 +5,40 @@
 import sys
 import singer
 from singer import metadata
-from tap_klaviyo.utils import get_incremental_pull, get_full_pulls, get_all_pages
+from tap_klaviyo.utils import get_incremental_pull, get_full_pulls, get_all_using_next
 
 LOGGER = singer.get_logger()
 
+API_VERSION = "2024-02-15"
+
 ENDPOINTS = {
-    'global_exclusions': 'https://a.klaviyo.com/api/v1/people/exclusions',
-    'lists': 'https://a.klaviyo.com/api/v1/lists',
-    'metrics': 'https://a.klaviyo.com/api/v1/metrics',
-    'metric': 'https://a.klaviyo.com/api/v1/metric/',
-    'campaigns': 'https://a.klaviyo.com/api/v1/campaigns'
+    'global_exclusions': 'https://a.klaviyo.com/api/profiles',
```

Comment:
Can you please provide the document reference to determin this API endpoint replacement?

Reply:
Yes, I'll do that.